### PR TITLE
Add more content to triage guideline

### PR DESCRIPTION
**Pull request by KershawChang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,18 +17,37 @@ Bugzilla
   - P5 bugs (would take): https://mzl.la/2hdnQXy
 
 Priorities
-- P1 bugs (Blocker: get someone assigned ASAP: also EMAIL necko@moz whenever we have a new P1 bug)
+- P1 bugs (Blocker: get someone assigned ASAP: let all necko folks know (by email or slack) whenever we have a new P1 bug)
 - P2 bugs (Active--we intend to work on these now or this quarter)
-- P3 bugs (backlog)
+- P3 bugs (backlog). P3 usually means no one will work on it, so we need to be careful
```
Comment (ddragana) [inline README.md:22]:
```suggestion
- P3 bugs (backlog). P3 usually means no one will work on it in the near future, so we need to be careful
```

File: README.md [hunk 2]
```diff
@@ -13,18 +17,37 @@ Bugzilla
...
+  - One exception is that we can mark a bug as P3 and also make it block another meta bug.
```
Comment (ddragana) [inline README.md:23]:
```suggestion
  - One exception is if we can mark a bug as P3 and also make it block another meta bug, such bugs may be resolved during a project or a focused cleanup, e.g. working on reducing fetch bugs.
```

File: README.md [hunk 3]
```diff
@@ -13,18 +17,37 @@ Bugzilla
...
 - There is no P4--don't use it
+  - This is used by bots.
 - P5 bugs (would take)
 
 Principles
 - P1 bugs MUST have a person assigned
 - Non-necko people can be assigned to necko bugs just fine
-- Bugs filed by wpt-bot should be marked triaged as P5. If something is needed there, someone can ni? one of us.
+- Bugs filed by wpt-bot are marked as P4 and not shown in the triage tool. If something is needed there, someone can ni? one of us
```
Comment (ddragana) [inline README.md:31]: This will change as per our discussion yesterday. But you can change that in a follow-up PR.

File: README.md [hunk 4]
```diff
@@ -13,18 +17,37 @@ Bugzilla
...
 - If you’re filing a bug you’ll be working on (or as part of a project you’re working on, even if you won’t be working directly on it), please triage appropriately, so as to not overload whoever is on triage that week :-D
 
 Mark as triaged
 - Set [necko-triaged] in the whiteboard for bugs that have been triaged.
-- Set Priority and Severity
+- If you know this bug belongs to another meta bug (e.g., QUIC, DoH, etc), remember to set it.
```
Comment (ddragana) [inline README.md:36]:
```suggestion
- If you know this bug belongs to a certain feature (e.g., QUIC, DoH, etc), remember to make it block the corresponding meta bug(s).
```

File: README.md [hunk 5]
```diff
@@ -13,18 +17,37 @@ Bugzilla
...
+- Set Priority and Severity:
+  - Severity only needs to be set when the bug type is `defect`. For other types, set to "N/A"
```
Comment (ddragana) [inline README.md:38]:
```suggestion
  - Severity only needs to be set when the bug type is `defect`. For other types, set the severity to "N/A"
```

File: README.md [hunk 6]
```diff
@@ -13,18 +17,37 @@ Bugzilla
...
+- More details
+  - If it's an urgent bug: Set it to P1 and ni? the owner or the manager to find an assignee.
```
Comment (ddragana) [inline README.md:40]: what you mean by the "owner"? the owner of a feature?

**Review by ddragana — COMMENTED** [review]
Looks good.

I have a couple of suggestion.

File: README.md (hunk 1 above)
Comment (KershawChang) [reply README.md:22]: Will do. Thanks.

File: README.md (hunk 6 above)
Comment (KershawChang) [reply README.md:40]: Yes, the owner of the feature.

**Pull request closed without merging**